runs/deleteRun.js

```diff
@@ -1,35 +1,41 @@
-import mongoose from 'mongoose'
-import RunModel from '../db/RunModel.js'
+import mongoose from "mongoose";
+import RunModel from "../db/RunModel.js";
 
-import resetDailyStats from '../dailyStats/resetDailyStats.js'
-import updatePlansFromRun from './updatePlansFromRun.js'
+import resetDailyStats from "../dailyStats/resetDailyStats.js";
+import updatePlansFromRun from "./updatePlansFromRun.js";
 
 // Removes the given run from the database
 const deleteRun = async (req, res) => {
   try {
-    const run = await RunModel.findById(req.params.id).lean()
+    const run = await RunModel.findById(req.params.id).lean();
 
     if (run == null) {
-      console.error(`Cannot delete run with id "${req.params.id}": id was not found.`)
-      return res.sendStatus(404)
+      console.error(
+        `Cannot delete run with id "${req.params.id}": id was not found.`,
+      );
+      return res.sendStatus(404);
     }
 
-    await new RunModel.deleteOne({_id: mongoose.Types.ObjectId(req.params.id)})
-    console.log(`Deleted run with id "${req.params.id}"`)
+    await RunModel.deleteOne({
+      _id: new mongoose.Types.ObjectId(req.params.id),
```

Comment:
This is the only line with a significant change. Will fix the eslint/prettier config and formatting on another branch.